java/kotlin-extractor/dev/wrapper.py

```diff
@@ -122,10 +122,13 @@ def forward(tool, forwarded_opts):
     if platform.system() == "Windows":
         tool = tool.with_suffix(".bat")
     assert tool.exists(), f"{tool} not found"
-    args = [tool]
-    args.extend(forwarded_opts)
-    ret = subprocess.run(args).returncode
-    sys.exit(ret)
+    cmd = [tool] + forwarded_opts
+    if platform.system() == "Windows":
+        # kotlin bat script is pretty sensible to unquoted args on windows
+        ret = subprocess.run(" ".join(f'"{a}"' for a in cmd)).returncode
```

Comment:
note for the reviewer: this was inspired by what we already do for windows in the internal repo `language-packs/java/tools/kotlin-extractor` wrapper script (external link [here](https://github.com/github/semmle-code/blob/a94fd7f44af928a0026d0a4cf1266ea209357829/language-packs/java/tools/kotlin-extractor#L136))